src/views/Dashboard.vue

```diff
@@ -190,6 +190,7 @@ export default {
 				})
 
 				if (importantRooms.length) {
+					// FIXME unread 1-1 conversations are not sorted like unread mentions in group chats
```

Comment:
```suggestion
					/** @todo FIXME unread 1-1 conversations are not sorted like unread mentions in group chats **/
```

What you think? Is best use the annotation and in a future we can search only by @toto to check all that we neet to pay attention and fix.